build.sbt

```diff
@@ -7,7 +7,7 @@ ThisBuild / scmInfo := Some(
   ScmInfo(url("https://github.com/avast/scala-server-toolkit"), "scm:git:git@github.com:avast/scala-server-toolkit.git")
 )
 
-ThisBuild / scalaVersion := "2.13.0"
+ThisBuild / scalaVersion := "2.12.10"
```

Comment:
We can start xcompiling, or is there a reason not to?

Reply:
The reason is stated in the description :) There is no stable version of http4s cross-built to both `2.12` and `2.13` so I think we can now be on `2.12` and cross-build to `2.13` once the ecosystem catches up.